lib/voxpupuli/release/rake_tasks.rb

```diff
@@ -52,3 +52,125 @@
     fail "Unable to find a CHANGELOG.md entry for the #{v} release."
   end
 end
+
+namespace :release do
+  desc "Prepare a release"
+  task prepare: ['release:porcelain:changelog'] do
+    v = Blacksmith::Modulefile.new.version
+    Rake::Task["strings:generate:reference"].invoke if File.exist?('REFERENCE.md')
+    puts <<~MESSAGE
+
+      Please review these changes and commit them to a new branch:
+
+          git checkout -b release-#{v}
+          git commit -m "Release #{v}"
+
+      Then open a Pull-Request and wait for it to be reviewed and merged).
+    MESSAGE
+  end
+
+  begin
+    require 'github_changelog_generator/task'
+    require 'puppet_blacksmith'
+  rescue LoadError
+    namespace :porcelain do
+      desc "Dummy"
+      task :changelog do
+        puts "Skipping CHANGELOG.md generation.  Ensure github_changelog_generator is present if you expected it to be generated."
+      end
+    end
+  else
+    class GCGConfig
```

Comment:
I think this class should live at the top level and be defined unconditionally. Otherwise any `Rakefile` that uses it will need to check if it's defined or fail hard when `github_changelog_generator` isn't present.